Sources/Public/Animation/LottiePlaybackMode.swift

```diff
@@ -7,111 +7,163 @@ import Foundation
+      _ fromFrame: AnimationFrameTime?,
+      toFrame: AnimationFrameTime,
+      loopMode: LottieLoopMode)
+
+    /// Plays the animation from a named marker to another marker.
+    ///
+    /// Markers are point in time that are encoded into the Animation data and assigned a name.
+    ///
+    /// NOTE: If markers are not found the play command will exit.
+    ///
+    /// - Parameter fromMarker: The start marker for the animation playback. If `nil` the
+    /// animation will start at the current progress.
+    /// - Parameter toMarker: The end marker for the animation playback.
+    /// - Parameter playEndMarkerFrame: A flag to determine whether or not to play the frame of the end marker. If the
+    /// end marker represents the end of the section to play, it should be to true. If the provided end marker
+    /// represents the beginning of the next section, it should be false.
+    /// - Parameter loopMode: The loop behavior of the animation.
+    case fromMarker(
+      _ fromMarker: String?,
+      toMarker: String,
+      playEndMarkerFrame: Bool = true,
+      loopMode: LottieLoopMode)
+
+    /// Plays the animation from a named marker to the end of the marker's duration.
+    ///
+    /// A marker is a point in time with an associated duration that is encoded into the
+    /// animation data and assigned a name.
+    ///
+    /// NOTE: If marker is not found the play command will exit.
+    ///
+    /// - Parameter marker: The start marker for the animation playback.
+    /// - Parameter loopMode: The loop behavior of the animation.
+    case marker(
+      _ marker: String,
+      loopMode: LottieLoopMode)
+
+    /// Plays the given markers sequentially in order.
+    ///
+    /// A marker is a point in time with an associated duration that is encoded into the
+    /// animation data and assigned a name. Multiple markers can be played sequentially
+    /// to create programmable animations.
+    ///
+    /// If a marker is not found, it will be skipped.
+    ///
+    /// If a marker doesn't have a duration value, it will play with a duration of 0
+    /// (effectively being skipped).
+    ///
+    /// If another animation is played (by calling any `play` method) while this
+    /// marker sequence is playing, the marker sequence will be cancelled.
+    ///
+    /// - Parameter markers: The list of markers to play sequentially.
+    case markers(_ markers: [String])
+  }
+
 }
 
 extension LottiePlaybackMode {
-  /// Plays the animation from the current progress to a progress value (0-1).
-  /// - Parameter toProgress: The end progress of the animation.
-  /// - Parameter loopMode: The loop behavior of the animation.
+  public static var paused: Self {
+    .paused(at: .currentFrame)
+  }
+
+  @available(*, deprecated, renamed: "LottiePlaybackMode.play(_:)", message: "Will be removed in a future major release.")
   public static func toProgress(_ toProgress: AnimationProgressTime, loopMode: LottieLoopMode) -> LottiePlaybackMode {
```

Comment:
Can we keep non-deprecated `LottiePlaybackMode.PlaybackMode.toProgress`, `.toFrame`, and `.toMarker` helpers? It looks like those were deprecated without a replacement

Reply:
@calda Should these be moved to `LottiePlaybackMode.PlaybackMode`?

Reply:
Yeah